api/app/utils/cffdrs.py

```diff
@@ -238,7 +237,9 @@ def total_fuel_consumption(  # pylint: disable=invalid-name
   #   FI:   Fire Intensity (kW/m)
 
 
-def head_fire_intensity(station: FBACalculatorWeatherStation,
+def head_fire_intensity(fuel_type: str,
```

Comment:
Why decompose into more parameters?

Reply:
Many reasons 1) I'd rather give the hfi function exactly what it needs - it's more clear what the inputs are that way. passing the entire object, one doesn't know what needs to be set or not (yes - I know some of those can be null - but it's just adding to more confusion 2) there was a circular dependency on that object, so I decided to rather just rip it out.